notebook.ipynb

```diff
@@ -109,7 +109,7 @@
     "from datetime import datetime, date, timedelta\n",
```

Comment:
<p>There is an error here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rawan-a-d/Air-Quality-Prediction/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='304'/>